src/Symfony/Component/Form/FormInterface.php

```diff
@@ -127,9 +148,9 @@ public function getData();
     /**
      * Returns the normalized data of the field.
      *
-     * @return mixed When the field is not submitted, the default data is returned.
+     * @return mixed when the field is not submitted, the default data is returned.
```

Comment:
These changes weren't supposed to happen? If I'm not mistaken, the first letter must be uppercase and the dot must be present.

Reply:
I see your commit `Apply FabPot.io fixes` and I am a little bit confused... Because I see capital letter everywhere after `@return` in Symfony...